### ISSUE TITLE
Intermittent consolidation issue

### ISSUE THREAD

**Issue by johnkerl** [single-cell-data/TileDB-SOMA#1670]
**Describe the bug**

During a consolidation operation, the Python process is dying with:
```
tiledb.cc.TileDBError: [TileDB::Reader] Error: Incorrect unfiltered tile size allocated.
```

This is intermittent with details to be provided by @atolopko-czi.

**To Reproduce**

TBD.

**Versions (please complete the following information):**

```
tiledbsoma.__version__        1.2.2
TileDB-Py tiledb.version()    (0, 21, 6)
TileDB core version           2.15.4
libtiledbsoma version()       libtiledb=2.15.1
```

(The census builder is intentionally down-revisioned from TileDB-SOMA latest.)

**Additional context**

TBD.

**Comment by johnkerl** [comment]
@atolopko-czi am I correct this has not recurred?

**Comment by bkmartinjr** [comment]
It has not, but please keep in mind that we have seen it twice, and have no evidence it is fixed.

**Comment by johnkerl** [comment]
@bkmartinjr @atolopko-czi on further analysis I believe this is resolved in core 2.17.2 (which is upcoming)

**Comment by johnkerl** [comment]
https://github.com/single-cell-data/TileDB-SOMA/tree/1.5.0 contains core 2.17.4
